src/resources/views/fields/toggle.blade.php

```diff
@@ -0,0 +1,103 @@
+<!-- radio toggle -->
+@php
+    $optionPointer = 0;
+    $optionValue = old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' ));
+@endphp
+
+<div @include('crud::inc.field_wrapper_attributes') >
+
+    <div>
+        <label>{!! $field['label'] !!}</label>
+    </div>
+
+    @if( isset($field['options']) && is_array($field['options']) )
+
+        @foreach ($field['options'] as $value => $label )
+            @php ($optionPointer++)
```

Comment:
Is the `$loop` variable supported in the versions of Blade we support? https://laravel.com/docs/5.4/blade#loops